about.html

```diff
@@ -0,0 +1,99 @@
+                    <li class="nav-item"><a class="nav-link" href="#portfolio">Portfolio</a></li>
+                    <li class="nav-item"><a class="nav-link" href="#contact">Contact</a></li>
+                </ul>
+            </div><button class="btn btn-primary" type="button">Join!</button>
+        </div>
+    </nav>
+    <header class="text-center text-white d-flex masthead header-section">
+        <div class="container my-auto">
+            <div class="row">
+                <div class="col-lg-10 mx-auto">
+                    <h1 class="text-uppercase"><strong class="text-end">Subaru sti owners club, nairobi, kenya!</strong></h1>
+                    <hr>
+                </div>
+            </div>
+            <div class="col-lg-8 mx-auto">
+                <p id="info-box">Car clubs have been a form of gathering car aficionados for many years that focus a passion for a certain type of car or driving activity.<a href="https://en.wikipedia.org/wiki/Car_club#cite_note-1">[1]</a>Historically, car clubs refer to off-line entities, typically organized as&nbsp;<a href="https://en.wikipedia.org/wiki/Non-profit_organization">non-profits</a>&nbsp;and run by volunteers (who were most often elected). Some clubs were large enough to be run as a paid business with salaried employees; in the 1960s, some were sponsored by&nbsp;<a href="https://en.wikipedia.org/wiki/Car_dealer">car dealers</a>.<a href="https://en.wikipedia.org/wiki/Car_club#cite_note-2">[2]</a>Many car clubs charge membership fees in exchange for benefits, such as publications and events. The publications typically contain photographs, messages from other members, service and parts advice, items and vehicles wanted and/or for sale, and historical material of interest to the membership. Car clubs often host gatherings (called "<strong>meets</strong>") which often also welcome interested non-members. Car clubs also may engage in other activities of various types, including&nbsp;<a href="https://en.wikipedia.org/wiki/Automobile_racing">races</a>,&nbsp;<a href="https://en.wikipedia.org/wiki/Cruising_(driving)">cruising</a>, shows,&nbsp;<a href="https://en.wikipedia.org/wiki/Car_modding">"mod" days</a>&nbsp;when garage equipment and service tools are available for members to perform and assist each other with&nbsp;<a href="https://en.wikipedia.org/wiki/DIY">DIY</a>&nbsp;work, or&nbsp;<a href="https://en.wikipedia.org/wiki/Community_service">community service</a>&nbsp;activities. Meets are basically a time for the club to hang out with one another and to talk about cars and do normal stuff, such as bowling, eating, movies, etc.</p>
+            </div>
+            <p id="contact-statement">Please contact us via email for any further questions about our car club! info@subaruowners.org</p>
+        </div>
+    </header>
+    <section id="about" class="bg-primary">
+        <div class="container">
+            <div class="row">
```

Comment:
- [ ] As I have explained below,  please uphold the general structure and design provided in behance:

Your design  | Behance design
------------- | -------------
<img width="1006" alt="Screenshot 2022-04-20 at 13 08 09" src="https://user-images.githubusercontent.com/77038610/164193323-cdb0843f-c2c0-4101-8815-eedfbb25e681.png"> | <img width="786" alt="Screenshot 2022-04-20 at 13 07 52" src="https://user-images.githubusercontent.com/77038610/164193260-9ac4f8bc-0c71-4e8c-8f70-15574bde4a3b.png">

>  Middle section should contain a centered column that includes a section header, a divider, a paragraph of text, and an image or large logo.

Reply:
Changed section to behance design.
<img width="867" alt="Screenshot 2022-04-21 at 15 06 18" src="https://user-images.githubusercontent.com/40231363/164454729-8d2d3836-3b64-41bf-a08b-b8d722c022e0.png">